src/ucp/core/ucp_ep.c

```diff
@@ -641,13 +641,34 @@ ucs_status_t ucp_ep_evaluate_perf(ucp_ep_h ep,
     ucp_worker_iface_t *wiface;
     uct_iface_attr_t *iface_attr;
     ucs_linear_func_t estimated_time;
+    double duration                         = 0;
+    ucs_status_t status;
 
     if (!ucs_test_all_flags(attr->field_mask,
-                            UCP_EP_PERF_ATTR_FIELD_ESTIMATED_TIME &
+                            UCP_EP_PERF_ATTR_FIELD_ESTIMATED_TIME) ||
+        !ucs_test_all_flags(param->field_mask,
```

Comment:
- no need to use `ucs_test_all_flags` for checking just one flag
- worth mentioning in the API description that at least message size is mandatory field. I'd also think about making op_type to be mandatory